File: morinocontest4/paiza6.py
p = int(input())

count = 0
count += p // 500
p %= 500
count += p // 100
p %= 100
count += p // 50
p %= 50
count += p // 10
p %= 10
count += p // 5
p %= 5
count += p // 1

print(count)

File: Brank/paiza6.swift
import Foundation

func countSubstring(searchStr: String, str: String) -> Int {
    var count = 0
    var startIndex = str.startIndex
    while let range = str.range(of: searchStr, range: startIndex..<str.endIndex) {
        count += 1
        startIndex = range.upperBound
    }
    return count
}


let searchStr = readLine()!
let str = readLine()!
print(countSubstring(searchStr: searchStr, str: str))